packages/design/src/FormManager/FormManagerLayout/TopNavigation.tsx

```diff
@@ -179,12 +179,10 @@ const MobileStepIndicator = () => (
       </span>
     </div>
     <div className="grid-col grid-col-8">
-      <select className="usa-select" name="options" id="options">
+      <select className="usa-select" name="options" id="options" defaultValue="value2">
```

Comment:
Just getting rid of an annoying react warning. Apparently, "defaultValue" is preferred to "selected" now